app/workers/terms_of_use/sign_up_service_updater_job.rb

```diff
@@ -5,6 +5,13 @@
 module TermsOfUse
   class SignUpServiceUpdaterJob
     include Sidekiq::Worker
+    sidekiq_options retry: 15 # 2.1 days using exponential backoff
```

Comment:
if we ever get upgraded to sidekiq 7 this is a new option:
```ruby
retry_for: 48.hours
```

https://github.com/sidekiq/sidekiq/blob/main/Changes.md#713